qutebrowser/browser/commands.py

```diff
@@ -709,6 +709,12 @@ def yank(self, what='url', sel=False, keep=False):
             sel = False
             target = "clipboard"
 
+        urlQuery = QUrlQuery(s)
```

Comment:
This should be `url_query` to conform with the usual naming of things in qutebrowser (except things coming from Qt)